WordPress/WordPressTest/TruthTable.swift

```diff
@@ -0,0 +1,21 @@
+/// Convenience to iterate over all possible combinations of multiple boolean values.
+enum TruthTable {
+
+    static let twoValues: [(Bool, Bool)] = [
```

Comment:
Something like this could be used for automatic value generation. Although if we need more than 3 or 4 it probably means the method we're testing has too many parameters, so it shouldn't be required.

```
struct TruthTable {
    static func values(_ n: Int) -> [[Bool]] {
        if n == 1 {
            return [[true], [false]]
        }

        let previousCombinations = values(n - 1)
        var result: [[Bool]] = []

        for combination in previousCombinations {
            result.append(combination + [true])
            result.append(combination + [false])
        }

        return result
    }
}
```

Reply:
Thank you for the suggestion. I decided to hard-code it because: 1) looked more like the text-book truth tables from my uni days, 2) the table for a given `n` is constant, so might has well save the computation.

For what is worth, I am considering extracting the logic in a package for the fun of it. If I were to do so, I would use the code you provided to code generate the tables for n = 2, 3, 4 etc., similar to what [Curry does](https://github.com/thoughtbot/Curry/blob/main/bin/generate).